src/clj/fluree/db/api.cljc

```diff
@@ -351,8 +351,7 @@
          (and query identity)
          (let [policy-db (<? (policy/wrap-identity-policy latest-db
                                                           identity
-                                                          (or (true? default-allow?)
-                                                              false)
+                                                          default-allow?
```

Comment:
This should be accompanied by a malli change that ensures only boolean values are include.

This changes means instead of having to be `true`, it only needs to but truthy... so even an empty string (`""`) or empty vector (`[]`) or anything other than `nil` would dramatically change the security characteristics of the ledger.

For things affecting security, we must ensure the most restrictive options are always taken until explicit intention is shown. This loosens that.